sema/harvest/service.py

```
@@ -34,11 +34,7 @@ class Harvest(ServiceBase):
     for all subjects in the configuration file.
     """
 
-    def __init__(
-        self,
-        config: str,
-        target_store_info: Optional[List[str]] = None,
-    ):
+    def __init__(self, config: str, *target_store_info: List[str]):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)